### PR TITLE
Nature Article

### PR DESCRIPTION
**Pull request by nickcall**
This commit is the first iteration of work which when complete will provide us with one article template capable of rendering either Nature, BMC or SpringerLink articles.

* Adds styles to be used in generic template (currently nature styling)

Work completed by @saramorueta @moddular @nickcall @abdallaomar

### CODE REVIEW

File: packages/nature-article/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (alexkilgour) [inline packages/nature-article/package-lock.json]: there are no dependencies, so need for this to be commited

File: packages/nature-article/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@springernature/nature-article",
+  "version": "0.0.1",
+  "description": "Frontend package for Nature articles"
```
Comment (alexkilgour) [inline packages/nature-article/package.json]: needs `"license": "MIT"` - hence your build failure

File: packages/nature-article/scss/50-components/article.scss [hunk 3]
```diff
@@ -0,0 +1,1563 @@
...
+// TODO: mobile first approach
+@media only screen and (max-width: 30em) {
+	.c-article-section__illustration-right,
+	.c-article-section__illustration-center {
+		margin: 0 auto 28px;
+		float: none;
+		max-width: 100%;
+	}
+}
+
```
Comment (alexkilgour) [inline packages/nature-article/scss/50-components/article.scss]: we have a media query helper in `context` and nature specific breakpoints

File: packages/nature-article/scss/50-components/article.scss [hunk 4]
```diff
@@ -0,0 +1,1563 @@
+/*
+Note: font-variant: small-caps doesn't display as we need for single character strings
+when source text is already uppercase, hence the use of font-size, text-transform
+*/
```
Comment (alexkilgour) [inline packages/nature-article/scss/50-components/article.scss]: I think we should put these into `context` now

File: packages/nature-article/package.json (hunk 2 above)
Comment (alexkilgour) [reply packages/nature-article/package.json]: also needs `nature-context` as a peerDependency

File: packages/nature-article/package-lock.json (hunk 1 above)
Comment (nickcall) [reply packages/nature-article/package-lock.json]: removed

File: packages/nature-article/package.json (hunk 2 above)
Comment (nickcall) [reply packages/nature-article/package.json]: added

File: packages/nature-article/scss/50-components/article.scss (hunk 3 above)
Comment (nickcall) [reply packages/nature-article/scss/50-components/article.scss]: will look at in future PRs

File: packages/nature-article/scss/50-components/article.scss (hunk 4 above)
Comment (nickcall) [reply packages/nature-article/scss/50-components/article.scss]: will look at in future PRs

File: packages/nature-article/scss/50-components/article.scss [hunk 5]
```diff
@@ -0,0 +1,1563 @@
...
+.u-small-caps {
+	font-size: 0.8em; /* scale based on containing text */
+	text-transform: uppercase;
+}
+
+.u-double-underline {
+	text-decoration: underline;
+	-webkit-text-decoration-skip: skip;
+	text-decoration-skip: ink;
```
Comment (ricky-lau) [inline packages/nature-article/scss/50-components/article.scss]: `text-decoration-skip: ink` has changed to `text-decoration-skip-ink: auto`

https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-skip-ink

**Pull request merged**